frontend/src/v4/routes/components/revisionsListItem/revisionsListItem.component.tsx

```diff
@@ -65,8 +65,8 @@ export const RevisionsListItem = (props: IProps) => {
 
 	const handleClick = (event) => props.onClick(event, props.data);
 	const handleToggleVoid = (event) => props.onToggleVoid(event, props.data);
-	const themeProps = useMemo(() =>
-		({ current, void: props.data.void, isPending }), [isPending, props.data.void, current]
+	const stateTheme = useMemo(() =>
+		({ current, isVoid: props.data.void, isPending }), [isPending, props.data.void, current]
 	);
```

Comment:
we can get rid of this and just pass the props to the container instead of using a memo, unless there is a good reason for using it.

Reply:
useMemo has been removed